core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.component.spec.ts

```diff
@@ -252,6 +252,9 @@ describe('Topics and Skills Dashboard Page', function() {
       expect(ctrl.select2DropdownIsShown).toBe(true);
     });
 
+    it('should return the upper count value for pagination', function() {
+      expect(ctrl.getUpperCountForPagination()).toEqual(1);
```

Comment:
Add more tests here to test the min() logic.

Reply:
Done.